### PR TITLE
AAAInternetPublishing.wtfast version 5.4.2.0

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/75303)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/labels/Validation-Unattended-Failed

> [ ] Have you tested your manifest locally with winget install --manifest <path>?

WindowsSandboxClient_rQK2LtnRv1

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/75303#pullrequestreview-1089801778

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
> Reply at [solomoncyj](https://matrix.to/#/@solomoncyj:matrix.org) 
> https://user-images.githubusercontent.com/15158490/187380803-a3bc9609-5e62-46e1-a40f-0709389eda00.png
> Can this be by passeb by /suspressmsgboxes?
>> I don't think so, because it's a dependency

**Comment by solomoncyj** [comment]
as long as the installation gets through, we should be clear, right?

**Comment by solomoncyj** [comment]
@denelon maybe dependencies? managed to install without installing dependencies by mods who wanted the dependencies to be resolved

File: manifests/a/AAAInternetPublishing/wtfast/5.4.2.0/AAAInternetPublishing.wtfast.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AAAInternetPublishing.wtfast
+PackageVersion: 5.4.2.0
```
Comment (ItzLevvie) [inline manifests/a/AAAInternetPublishing/wtfast/5.4.2.0/AAAInternetPublishing.wtfast.installer.yaml:5]:
```suggestion
PackageVersion: 5.4.2.0
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x86
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/AAAInternetPublishing/wtfast/5.4.2.0/AAAInternetPublishing.wtfast.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AAAInternetPublishing.wtfast
+PackageVersion: 5.4.2.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerSwitches:
+    Custom: /suppressmsgboxes
```
Comment (ItzLevvie) [inline manifests/a/AAAInternetPublishing/wtfast/5.4.2.0/AAAInternetPublishing.wtfast.installer.yaml:10]:
```suggestion
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/a/AAAInternetPublishing/wtfast/5.4.2.0/AAAInternetPublishing.wtfast.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: AAAInternetPublishing.wtfast
+PackageVersion: 5.4.2.0
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VC++2015-2022Redist-x86
+Installers:
+- Architecture: x64
+  InstallerType: inno
+    InstallerSwitches:
+    Custom: /suppressmsgboxes
```
Comment (ItzLevvie) [inline manifests/a/AAAInternetPublishing/wtfast/5.4.2.0/AAAInternetPublishing.wtfast.installer.yaml:13]:
```suggestion
  InstallerSwitches:
    Custom: /SUPPRESSMSGBOXES
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**